__tests__/components/dashtable_v2.test.jsx

```diff
@@ -0,0 +1,161 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import GlobalDashboardTable from '../../components/dashtable_v2.js';
+
+describe('GlobalDashboardTable', () => {
+  test('renders dashtable with correct data format', () => {
+    const classroomId = 'cljbqrfoa0006kohk894ym6v3';
+
+   
+    const studentData = [
```

Comment:
It might be helpful to store the hardcoded data in a separate file and then import that data into this test case.  In this way, we can reuse the same data files across multiple tests.

Reply:
Hi @utsab, we applied your feedback and the tests are successfully passing. I will update the PR with the new test shortly. 

You mentioned to store the hardcoded data in a separate file, is there a specific folder this hardcoded data should be under, and should it have a specific name. We named it testing-data.jsx, and have it under testing-data folder.

Do you have any feedback on our current approach, or should we explicitly be getting the response from the mock API and using that as the data?

Additionally, if you have resources on how this is typically done please feel free to share, thanks!